### DISCUSSION TITLE
Text error when trying to play multiplayer without being logged in

### DISCUSSION THREAD

**Discussion by Jan-Arbus** [ppy/osu#17866] [Q&A] [answered]
Steps to reproduce
1. Make sure you are not logged in
2. Tap the multiplayer button
![屏幕截图 2022-04-18 201510](https://user-images.githubusercontent.com/74862110/163807177-9949a6cc-70ec-4955-8573-693b1ea946e1.png)

**Comment by respektive** [comment, marked as answer]
Well, you gotta be online to play multiplayer? I don't understand. Or do you think the text should be more clear that you have to be logged in instead of just online?

**Reply by Jan-Arbus** [reply]
I mean the prompt text is misspelled
Multiplayer is spelled multi 'yo
sorry, my English is not well ;(

**Reply by bdach** [reply]
there's nothing inherently wrong with the message. it's a bit informal but that's all.
